src/app/pages/poll/poll.component.ts

```diff
@@ -100,6 +101,25 @@ export class PollComponent {
     });
   }
 
+  /**
+   * Update the current poll item for this poll on the server
+   *
+   * @param event Click event
+   * @param pollItemId Id of the poll item
+   */
+  onRunPollItem(event: any, pollItemId: number): void {
+    event.stopPropagation();
+    const oldPollItem = this.poll.currentItem;
+    this.poll.currentItem = this.loadingRunPollItemId = pollItemId === this.poll.currentItem ? null : pollItemId;
+    this.pollService.update(this.poll).subscribe((_) => {
+      this.loadingRunPollItemId = -1;
```

Comment:
What values can this variable take? Only `0` and `-1`?

Reply:
As you see in line 113, this.runLoadingPollItemId holds an id of a poll item. Thus, not only 0 and -1.

Reply:
ah ok, missed that

Reply:
because of the double assignment 😅